docs/integration-handbook.md

```diff
@@ -0,0 +1,246 @@
+The following topics are made available:
+
+- `/fsds/gps` [sensor_msgs/NavSatFix](https://docs.ros.org/api/sensor_msgs/html/msg/NavSatFix.html)
+GPS messages. [Read all about the gps model here](gps.md).
+
+- `/fsds/camera/CAMERA_NAME` [sensor_msgs/Image](https://docs.ros.org/api/sensor_msgs/html/msg/Image.html)
+One of this topic type will exist for every camera specified in the `settings.json` file.
+On this topic, camera frames are published. The format will be bgra8. 
+`CAMERA_NAME` will be replaced by the corresponding in the `Cameras` object in the `settings.json` file.
+
+- `/fsds/camera/CAMERA_NAME/camera_info` [sensor_msgs/CameraInfo](https://docs.ros.org/api/sensor_msgs/html/msg/CameraInfo.html)
+This topic publishes metadata about the related camera.
+For every frame sent on `/fsds/CAMERA_NAME` 1 message will be sent on this topic.
+
+- `/fsds/imu` [sensor_msgs/Imu](https://docs.ros.org/api/sensor_msgs/html/msg/Imu.html)
+IMU messages. [Read all about the IMU model here](imu.md).
+
+### Signal topics
+The following topics indicate the beginning and the end of the mission:
+
+- `/fsds/signal/go` [fsds_ros_bridge/GoSignal](../ros/src/fsds_ros_bridge/msg/GoSignal.msg)  
+  GO signal that is sent every second by the ROS bridge. The car is only allowed to drive once this message has been received. If no GO signal is received for more than 4 seconds, the AS can assume that `fsds_ros_bridge` has been shut down. This message also includes the mission type and track.
+
+- `/fsds/signal/finished` [fsds_ros_bridge/FinishedSignal](../ros/src/fsds_ros_bridge/msg/FinishedSignal.msg)  
+  Finished signal that is sent by the AS to stop the mission. The `fsds_ros_bridge` will shut down when this message is received.
+
+### Vehicle setpoints
+Publishing on the following topic controls the vehicle:
+
+- `/fsds_ros_bridge/control_command` [fsds_ros_bridge/ControlCommand](../ros/src/fsds_ros_bridge/msg/ControlCommand.msg)
+
+This message includes the dimensionless values throttle, steering and brake. 
+Throttle and brake range from 0 to 1.
+For steering `-1` steers full to the left and `+1` steers full to the right.
+
+
+### Transforms
+//todo: We have to figure out how these transforms are working.
+
+- `/tf` [tf2_msgs/TFMessage](https://docs.ros.org/api/tf2_msgs/html/msg/TFMessage.html)
+
+## Vehicle dynamic model
+The vehicle dynamic model is a third-party high-fodelity model and will be the same for all teams. More details and information on this choice can be found [here](vehicle_model.md).
+
+## 3D vehicle model
+//todo: reference vehicle model documentation in the open PR
+This chapter will describe how to change the 3d model of the vehicle and how to provide the 3d model for usage during the competition. 
+At this moment we have no idea how this works sooooo when we figure it out this will be filled in.
+
+## Competition deployment
+A few weeks before the competition, each team will receive the ssh credentials to an Ubuntu google cloud instance.
```

Comment:
Wait are we going to give to the competitors Ubuntu or Windows machines?

Reply:
Ubuntu! Teams' will run their AS in Ubuntu while the simulation runs on Windows.